StandaloneMmPkg/Core/StandaloneMmCore.c

```diff
@@ -679,7 +679,10 @@ StandaloneMmMain (
   //
   MmramRangesHob = GetNextGuidHob (&gEfiMmPeiMmramMemoryReserveGuid, HobStart);
   if (MmramRangesHob == NULL) {
-    return EFI_UNSUPPORTED;
+    MmramRangesHob = GetFirstGuidHob (&gEfiSmmSmramMemoryGuid);
+    if (MmramRangesHob == NULL) {
+      return EFI_UNSUPPORTED;
+    }

```

Comment:
gEfiSmmSmramMemoryGuid is defined in PI spec but the gEfiMmPeiMmramMemoryReserveGuid is not. So, please locate the spec-defined GUIDed HOB first.